Source/sg_EditSpeakersWindow.hpp

```diff
@@ -28,6 +28,68 @@ namespace gris
 class EditableTextCustomComponent;
 class MainContentComponent;
 class GrisLookAndFeel;
+//==============================================================================
+// LabelTextEditorWrapper
+
+// Helper struct to trigger static_assert for unsupported types
+template<typename T>
+struct always_false : std::false_type {
+};
+
+struct LabelWrapper {
+    LabelWrapper(GrisLookAndFeel & lookAndFeel);
```

Comment:
constructors with only one argument should really really be marked explicit to avoid bad surprises: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rc-explicit